### DISCUSSION TITLE
How to determine that Btrace script has successfully intruded into the code？

### DISCUSSION THREAD

**Discussion by jndxcaiwei** [btraceio/btrace#570] [Q&A]
I use the btrace script in the automated test with Linux environment. 
One thread is used to run the btrace script, and the other thread performs some operations on the application to trigger the btrace. However, it is found that whether the btrace has successfully intruded into the code cannot be judged by if the PID of the btrace script process  exists. 
The time of each btrace intrusion into the code is inconsistent. Is there any identification indicating that the btrace has successfully worked？
Thanks a lot to your  replay.

**Comment by jbachorik** [comment]
You could use eg. `btrace.port` system property to check whether the Btrace agent was loaded and started with the socket server.

There is no special 'event' for when a class is instrumented, I am afraid.
